src/api/auth.js

```diff
@@ -128,6 +128,45 @@ export function upgradeFacilitatorAccessToken(facilitatorAccessToken : string, {
     });
 }
 
+export function upgradeFacilitatorAccessTokenWithIgnoreCache(facilitatorAccessToken : string, buyerAccessToken : ?string, orderID : string) : ZalgoPromise<string> {
+    return inlineMemoize(upgradeFacilitatorAccessTokenWithIgnoreCache, () => {
+        clearLsatState();
+        onLsatUpgradeCalled();
+        const headers = {};
+        if (buyerAccessToken) {
+            headers[HEADERS.ACCESS_TOKEN] = buyerAccessToken;
+          }
```

Comment:
At this point the buyer access will be available. We can remove this if condition. 
```suggestion
          headers[HEADERS.ACCESS_TOKEN] = buyerAccessToken;
```

Reply:
type OrderActionOptions = {|
    orderID : string,
    paymentID : ?string,
    payerID : ?string,
    restart : () => ZalgoPromise<void>,
    intent : $Values<typeof INTENT>,
    facilitatorAccessToken : string,
    buyerAccessToken : ?string,
    partnerAttributionID : ?string,
    forceRestAPI : boolean,
    onError : OnError,
    experiments: Experiments
|};

function buildOrderActions({ intent, orderID, restart, facilitatorAccessToken, buyerAccessToken, partnerAttributionID, forceRestAPI, onError, experiments } : OrderActionOptions) : OrderActions {
    const get = memoize(() => {
        if (experiments?.upgradeLSATWithIgnoreCache) {
            return upgradeFacilitatorAccessTokenWithIgnoreCache(facilitatorAccessToken, buyerAccessToken, orderID)
   

buyerAccessToken is optional, so when we pass it to upgradeFacilitatorAccessTokenWithIgnoreCache, the value could be undefined or null.

Reply:
We should make it mandatory. Without this data the mutation will fail.